src/test_helpers/integration.cljs

```diff
@@ -59,4 +78,143 @@
 
 (defn log-headline
   [test-name]
-  (log/info (str "========= " (name test-name) " ==================")))
+  (log/info (str "==== " test-name " ====")))
+
+(defn wait-for
+  "Returns a promise that resolves when all `event-ids` are processed by re-frame,
+  otherwise rejects after `timeout-ms`.
+
+  If an event ID that is expected in `event-ids` occurs in a different order,
+  the promise will be rejected."
+  ([event-ids]
+   (wait-for event-ids default-re-frame-wait-for-timeout-ms))
+  ([event-ids timeout-ms]
+   (let [waiting-ids (atom event-ids)]
+     (js/Promise.
```

Comment:
can use promesa here, but doesn't make a difference really

Reply:
Thanks @clauxx I forgot to update this one because I added Promesa separately, after I saw how it all looked without it.